### ISSUE TITLE
feature request: more information on "Manuscripts" landing page

### ISSUE THREAD

**Issue by annamorphism** [DDMAL/cantus#639]
As we get more manuscripts into Cantus Ultimus, we will want some way of easily telling what the sources are without having to open them--i.e. having some sort of description available on the landing page.
This could be either a column with a snippet of the "short description" (how CDB handles it), or--maybe prettier--some sort of hover-over that gives the Manuscript info (as we have it on the pages for the manuscripts themselves).

It might also be nice to be able to sort the columns of name/date information, to easily find "something from the 13th century" or "that manuscript from Tours" without having to run a search.  And it would be *really* nice to be able to turn up all the graduals or all the antiphoners--for some reason even the search function doesn't seem to find this.

**Comment by fujinaga** [comment]
Excellent suggestions!

**Comment by dchiller** [comment]
Great suggestion! And all eminently doable. 

@annamorphism When you reference these manuscripts "genres" (graduals/antiphoners), it looks like that information is available in the "summary" text on CantusDB (for example, I'm looking at the entries at https://cantus.uwaterloo.ca/sources), but there isn't a specific field that includes just that. Is that right?

**Comment by annamorphism** [comment]
Correct---there is a specific field for centuries and one for geography (and one for monastic/not-monastic), but the "genre" is hiding in the text description (probably a feature to add one day...). The summary usually contains all of this, including the genre, in text.

**Comment by dchiller** [comment]
Excellent, thanks!
